server/src/main/scala/com/ossuminc/riddl/lsp/server/RiddlLSPTextDocumentService.scala

```diff
@@ -0,0 +1,25 @@
+package com.ossuminc.riddl.lsp.server
+
+import org.eclipse.lsp4j.jsonrpc.messages
+import org.eclipse.lsp4j.{CompletionItem, CompletionList, CompletionParams, DidChangeTextDocumentParams, DidCloseTextDocumentParams, DidOpenTextDocumentParams, DidSaveTextDocumentParams}
+import org.eclipse.lsp4j.services.TextDocumentService
+
+import java.util
+import java.util.concurrent.CompletableFuture
+
+object RiddlLSPTextDocumentService extends TextDocumentService {
```

Comment:
Shouldn't this be a class so that it can be instantiated for the return value for getTextDocumentService in RiddlLSPServer ?